packages/client/src/cards.tsx

```diff
@@ -0,0 +1,23 @@
+import { range } from 'lodash';
+
+export const SUIT = {
+  Denari: 'denari',
```

Comment:
Can we use the English names for these? Same for numbers.

Reply:
I think those are italian names not sure the play well if we translate. I copied the structure from someone else but I think it's a good idea for this game

Reply:
But it's just a 1:1 translation from the cards, right? The names are not unique to the game, or unique enough, they are just the card numbers and symbols. You can just take some cards off, from a common deck and play the game.
If we put Italian text like that, I think it's mostly going to be targeting Italians. It is too much to ask from non-Italians to learn/remember.